### ISSUE TITLE
Worth keeping?

### ISSUE THREAD

**Issue by wxwilcke** [SPINLab/minecraft#1]
Is this repo worth keeping, especially since it is just only simple script and over 4 years old?

**Comment by ndkv** [comment]
Nope. I'm sure Brian (who wrote the code) has a copy of this script somewhere. 


On 9 April 2018 at 13:10:52, Xander Wilcke (notifications@github.com) wrote:

Is this repo worth keeping, especially since it is just only simple script and over 4 years old?
